Ecommerce/index.html

```diff
@@ -42,5 +42,31 @@ <h1>Contact us</h1>
     <textarea name="myMessageBox" rows="4"></textarea>
     <button>submit</button>
  </form>
+
+
+ <table>
+    <thead>
+        <tr>
+            <th>Company</th>
+            <th>Model</th>
+            
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>Dodge</td>

```

Comment:
The closing tag for the 'tr' element should be on a separate line for better readability.